test/tests/swap_case.ts

```diff
@@ -0,0 +1,105 @@
+/* globals describe, it */
+import TestCase from '../testcase';
+
+describe('swapping case', function() {
+  it('should swap case at cursor and moving cursor to the right', async function() {
+    const t = new TestCase(['oo']);
+    t.sendKeys('0');
```

Comment:
the 0 at the start of these test cases is generally unnecessary.  it's perhaps a bit weird to assume, but ive been doing it elsewhere